src/Telegram.Bot/TelegramBotClientExtensions.ApiMethods.cs

```diff
@@ -322,6 +325,7 @@ await botClient.ThrowIfNull(nameof(botClient))
         int? replyToMessageId = default,
         bool? allowSendingWithoutReply = default,
         IReplyMarkup? replyMarkup = default,
+        int? messageThreadId = default,
```

Comment:
we try to have the same parameters order as they are described in docs. thus `messageThreadId` should be the first optional parameter here.
or even keep it in place with some hacky approach:
`public static void Foo([Optional] [DefaultParameterValue(null)] object? x, string y) {}`